pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -440,6 +440,15 @@ func updatePodFromAllocation(pod *v1.Pod, allocated state.PodResourceInfo) (*v1.
 	}
 
 	updated := false
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
```

Comment:
Still hoping this comment can be addressed: https://github.com/kubernetes/kubernetes/pull/132919#discussion_r2487604622